src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -471,13 +485,15 @@ This means you should very likely pin the platform ${chalk.greenBright(
 ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
     }
 
+    // TODO What and why?
```

Comment:
This is setting the executable flag on non Windows platforms I think.
Does it makes sense to do that _all the time_ and not only if a specific error is raised? I would assume this has a (small) performance impact.

Reply:
```suggestion
    // Make executable on non Windows platforms
```